player-small.js

```diff
@@ -99,19 +99,11 @@ var CPlayer = function() {
             }
 
             // Oscillator 1
-            t = o1t;
-            if (o1xenv) {
-                t *= e * e;
-            }
-            c1 += t;
+            c1 += o1t * Math.pow(e,o1xenv);
```

Comment:
Given that there is now [wide adoption of the exponent operator](https://caniuse.com/#search=Exponentiation%20operator), I think it makes sense to use `**` instead of `Math.pow`. You could add a comment about how to support older browsers by using `Math.pow` instead..